src/stdpar/DataFormats/PixelErrors.h

```diff
@@ -10,10 +10,10 @@
 // Better ideas for the placement of these?
 
 struct PixelErrorCompact {
-  uint32_t rawId;
-  uint32_t word;
-  uint8_t errorType;
-  uint8_t fedId;
+  uint32_t rawId {0};
+  uint32_t word {0};
+  uint8_t errorType {0};
+  uint8_t fedId {0};
```

Comment:
It was mostly to play safe as I did not check how this particular struct was used. If it's written to by the device before being read and you noticed important cost I'll revert it back.

Reply:
For this particular struct the initialization likely has negligible impact. I was more commenting about the general approach. Could you undo the added initializations elsewhere as well (since you already started that route :))?

Reply:
Done except for Counters where it was memset to 0 before use.